codecov.yml

```diff
@@ -1,2 +1,3 @@
 ignore:
   - "app/controllers"
+  - "config/initializers/scheduler.rb"
```

Comment:
@JoshDevHub 
codecov started complaining after I appended the “s” after the namespaces in the scheduler
Is the scheduler something we need to test?

Reply:
Nah we shouldn't have to test anything happening in the scheduler.